src/event.js

```diff
@@ -610,8 +610,7 @@ jQuery.event = {
 			event,
 			{
 				type: type,
-				isSimulated: true,
-				originalEvent: {}
```

Comment:
> What docs? This is for a simulated event, I don't believe there's anything special in the docs about this.

See above.

> There's no possible use case to be back compatible with. Did someone provide an actual scenario that I missed?

This is exposed documented API, according to the semver this would be a breakage, therefore it could break real code even if it is an empty object.

> You're concerned about the delete over setting to an empty object in terms of logic and bytes?

See alternative pulls.

> That's impossible, considering it's been an empty object forever.

My argument not about past but about future - see alternative pulls

> And are the properties legitimate considering they're not even from the correct event type?

Why not?

> Consistency with what?

With other events and with other environments where we don't need to shim those events